scripts/ci-entrypoint.sh

```diff
@@ -220,7 +220,8 @@ if [[ -n "${TEST_CCM:-}" ]]; then
 --set cloudControllerManager.replicas="${CCM_COUNT}" \
 --set cloudControllerManager.enableDynamicReloading="${ENABLE_DYNAMIC_RELOADING}"  \
 --set cloudControllerManager.cloudConfig="${CLOUD_CONFIG}" \
---set cloudControllerManager.cloudConfigSecretName="${CONFIG_SECRET_NAME}"
+--set cloudControllerManager.cloudConfigSecretName="${CONFIG_SECRET_NAME}" \
+--set cloudControllerManager.logVerbosity=4
```

Comment:
Should we create a variable for this like `CCM_LOG_VERBOSITY` that defaults to 4?
`export CCM_LOG_VERBOSITY="${CCM_LOG_VERBOSITY:-4}"`

This is a trivial change, but it's odd to have this be the only hard-coded parameter in the argument list.

Reply:
Or we can change the default value here https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/helm/cloud-provider-azure/values.yaml#L30
Which one do you prefer?

Reply:
Do we want more logging in general from CCM, or just in these e2e tests?

If we want it in general, then changing the default in cloud-provider-azure makes sense. Otherwise, let's do it here in CAPZ, but maybe create `CCM_LOG_VERBOSITY` for consistency.

Reply:
Done.